src/node-data/nodes.json

```
@@ -8,25 +8,41 @@ onLoadMarkers(
     "status": "live",
     "device_count": 2
   }, {
-    "lat": 40.425660,
+    "lat": 40.425660,40.422555, -79.990967
     "lon": -79.983210,
```

```suggestion
    "lat": 0.422555, 
    "lon": -79.990967,
```

---

But looking at this, I think you've accidentally copypasted the _next_ location's coordinates, so I think you actually want to roll back the change:

```suggestion
    "lat": 40.425660,
    "lon": -79.983210,
```